### ISSUE TITLE
Live Stream

### ISSUE THREAD

**Issue by hamdanfadi** [matijse/eufy-ha-mqtt-bridge#55]
My I know if possible when I click on the camera to watch live stream? Is this possible?

**Comment by matijse** [comment]
This might be possible, but not right now. I am (slowly) changing the code for this script to use a new library (https://github.com/bropat/eufy-security-client) with a lot more features, so it might be possible in the future. 

The creator of that library has also created an integration for io.broker, but it is also possible to use that in Home Assistant. More information on that is in this topic  https://community.home-assistant.io/t/eufy-camera-integration/121758/1017

**Comment by hamdanfadi** [comment]
My I know if you have started with this feature or not?

**Comment by matijse** [comment]
No, I've started the rebuild and after that is done I will check if/how I can integrate live streaming... You can follow the rebuild on the `rewrite` branch.

**Comment by umutcelebi** [comment]
Any updates?
